.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
This should just be in your global .gitignore (https://help.github.com/articles/ignoring-files).